### ISSUE TITLE
Full path to output directory during "run command"

### ISSUE THREAD

**Issue by jcook793** [Swoffa/SublimeKickAssemblerC64#64]
The latest version of VICE for macOS seems to have a bug loading `.prg` files from relative paths. That is, this works:
```
x64sc /Users/jcook/personal/test-kickassembler/bin/GoatPowerExample.prg
```
but using a relative path doesn't load anything:
```
x64sc bin/GoatPowerExample.prg
```

I attempted to work around this problem by changing `kickass_run_command_x64`, but I can't seem to figure out any available environment variables for the fully-qualified path to the current project directory or the output directory. Are there any available during the run command that could help with this? Thank you!

**Comment by Swoffa** [comment]
Thanks @jcook793 for creating an issue!
I also noticed this, but had no time to investigate. Good to get confirmation. Will be fixed.

**Comment by Swoffa** [comment]
@jcook793 as a workaround, I think you can use this for `kickass_run_command_x64`:
```
${kickass_run_path}\" -logfile \"${file_path}/${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${file_path}/${kickass_output_path}/${build_file_base_name}.vs\" ${kickass_run_args} \"${file_path}/${kickass_output_path}/${start_filename}
```
Like this in the settings json:
```
"kickass_run_command_x64": "\"${kickass_run_path}\" -logfile \"${file_path}/${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${file_path}/${kickass_output_path}/${build_file_base_name}.vs\" ${kickass_run_args} \"${file_path}/${kickass_output_path}/${start_filename}\"",
```

**Comment by Swoffa** [comment]
...and here is a workaround for the debug command:
```
"kickass_debug_command_x64": "\"${kickass_debug_path}\" -logfile \"${file_path}/${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${file_path}/${kickass_output_path}/${build_file_base_name}_MonCommands.mon\" ${kickass_debug_args} \"${file_path}/${kickass_output_path}/${start_filename}\""
```

**Comment by jcook793** [comment]
Ahhhhh - `${file_path}` is the secret sauce, that worked great! Thank you so much for the quick follow-up and for this great Sublime Text package!

**Comment by Swoffa** [comment]
@jcook793 A fix has now been released, in version 1.3.4. \o/
